### PR TITLE
Backported non-breaking fixes and updates from main branch

### PR DESCRIPTION
**Pull request by hfp**
* tool_pexec.sh: more portable use of "time" command.
* tool_test.sh: select specific node (SRUN_NODE).
* Updated performance report scripts.
* Adjusted code format (minor).
* Fixed typos (code comments).
* Test LIBXSMM_SQRTF.

### CODE REVIEW

File: src/generator_gemm_amx.c [hunk 1]
```diff
@@ -1256,7 +1256,7 @@ void libxsmm_generator_gemm_init_micro_kernel_config_tileblocking(libxsmm_gemm_d
   } else if (LIBXSMM_DATATYPE_I8 == LIBXSMM_GETENUM_INP( i_xgemm_desc->datatype )) {
     k_blocking = 64;
   } else {
-    /* shouldn't happen */
+    /* should not happen */
```
Comment (alheinecke) [inline src/generator_gemm_amx.c:1259]: We should not do code clean-ups in this file

File: src/generator_gemm_amx_emu.c [hunk 2]
```diff
@@ -1498,7 +1498,7 @@ void libxsmm_generator_gemm_amx_kernel_emu( libxsmm_generated_code*        io_ge
       /* C pointer */
       libxsmm_x86_instruction_alu_mem( io_generated_code, l_micro_kernel_config.alu_mov_instruction,
                                        i_gp_reg_mapping->gp_reg_help_1, LIBXSMM_X86_GP_REG_UNDEF, 0, 96, i_gp_reg_mapping->gp_reg_c, 0 );
-      /* batch reduce count & offsett arrays*/
```
Comment (alheinecke) [inline src/generator_gemm_amx_emu.c:1501]: we should not do cleanups in this file

**Review by alheinecke — CHANGES_REQUESTED** [review]
we should not do unnecessary changes related to AMX

File: src/generator_gemm_amx_emu.c (hunk 2 above)
Comment (hfp) [reply src/generator_gemm_amx_emu.c:1501]: ACK.

File: src/generator_gemm_amx.c (hunk 1 above)
Comment (hfp) [reply src/generator_gemm_amx.c:1259]: ACK.

**Comment by hfp** [comment]
AMX related changes are backed out and remain with `main` branch. Main reason for this PR is `scripts/tool_pexec.sh` failed on our Jetson installation.

**Comment by hfp** [comment]
Test `unary_reduce_max_cols_idx_i64_argop1_x_overwrite_f32_eqld` failed on the Jetson. Needs separate investigation (PR will be merged as unrelated).

**Pull request merged**